src/config.rs

```diff
@@ -2207,16 +2207,13 @@ impl Default for BackupConfig {
     }
 }
 
-// TODO: `CdcConfig` is used by both `cdc` worker and `resolved ts` worker
-// should separate it into two configs
-#[derive(Clone, Serialize, Deserialize, PartialEq, Debug, Configuration)]
+#[derive(Clone, Serialize, Deserialize, PartialEq, Debug)]
 #[serde(default)]
 #[serde(rename_all = "kebab-case")]
 pub struct CdcConfig {
```

Comment:
Why keeps the struct?

Reply:
There are configs only use in cdc

Reply:
Then why remove `Configuration` derive?

Reply:
`Configuration` trait is used for config that can be changed dynamically, `CdcConfig` is not supported to doing so currently.